src/repositories/DocumentGroupRepository.ts

```diff
@@ -1,15 +1,15 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import type { DocumentGroupApi, FrontifyDocumentGroup } from '../types';
+import type { DocumentGroup, DocumentGroupApi } from '../types';
 import { HttpClient } from '../utilities/httpClient';
 import { mapDocumentApiToFrontifyDocument } from './DocumentRepository';
 
-export const getDocumentGroupsByProjectId = async (projectId: number): Promise<FrontifyDocumentGroup[]> => {
+export const getDocumentGroupsByProjectId = async (projectId: number): Promise<DocumentGroup[]> => {
     const { result } = await HttpClient.get<DocumentGroupApi[]>(`/api/documentgroup?project_id=${projectId}`);
     return result.data.map(mapDocumentGroupApiToFrontifyDocumentGroup);
 };
 
-const mapDocumentGroupApiToFrontifyDocumentGroup = (documentGroupApi: DocumentGroupApi): FrontifyDocumentGroup => {
+const mapDocumentGroupApiToFrontifyDocumentGroup = (documentGroupApi: DocumentGroupApi): DocumentGroup => {
```

Comment:
Should update method name as well:
```suggestion
const mapDocumentGroupApiToDocumentGroup = (documentGroupApi: DocumentGroupApi): DocumentGroup => {
```